stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2042,14 +2042,13 @@ end
 
 @testset "check buffers" for n in 1:3
     local A
-    colptr = [1,2,3,4]
     rowval = [1,2,3]
     nzval1  = Int[]
     nzval2  = [1,1,1]
-    A = SparseMatrixCSC(n, n, colptr, rowval, nzval1)
```

Comment:
Actually, this test seems bogus altogether (it's explicitly constructing a bad matrix). I missed the point initially, but I would just eliminate this test (and add asserts on construction forcing consistency between buffers). I would just remove it.

Reply:
I just read through the discussion in https://github.com/JuliaLang/julia/pull/20464 to refresh my memory. The symbolic factorizations in SuiteSparse don't reference the `nzval` buffer so this might be related to that. I don't remember if any of `SuiteSparse` wrappers currently make use of this feature. We should check that. We might have allowed an empty `nzval` such that people could do symbolic factorizations without the non-zeros but that might not be a good idea to allow such invalid instances of `SparseMatrixCSC`. Expert users should be able to work directly with `CHOLMOD.Sparse` for that.